localstack/services/stepfunctions/asl/antlr/ASLIntrinsicParser.g4

```diff
@@ -31,19 +31,16 @@ state_fun_name
     | MathAdd
     | StringSplit
     | UUID
-    ;
+;
 
-func_arg_list
-    : LPAREN func_arg (COMMA func_arg)* RPAREN
-    | LPAREN RPAREN
-    ;
+func_arg_list: LPAREN func_arg (COMMA func_arg)* RPAREN | LPAREN RPAREN;
 
-func_arg
-    : STRING                    #func_arg_string
-    | INT                       #func_arg_int
-    | NUMBER                    #func_arg_float
-    | (TRUE | FALSE)            #func_arg_bool
-    | CONTEXT_PATH_STRING       #func_arg_context_path
-    | JSON_PATH_STRING          #func_arg_json_path
-    | func_decl                 #func_arg_func_decl
-    ;
+func_arg:
+    STRING                # func_arg_string
+    | INT                 # func_arg_int
+    | NUMBER              # func_arg_float
+    | (TRUE | FALSE)      # func_arg_bool
+    | CONTEXT_PATH_STRING # func_arg_context_path
+    | JSON_PATH_STRING    # func_arg_json_path
+    | states_func_decl    # func_arg_func_decl
+;
```

Comment:
Are EOF newlines are possible to validate/unify with the formatter?

Reply:
Unfortunately I don't see an options to achieved this